workspaces/host/Makefile

```diff
@@ -9,14 +9,26 @@
 # See the `LICENSING.markdown` file in the Veracruz root directory for
 # licensing and copyright information.
 
-.PHONY: all clean clean-cargo-lock clean-datasets datasets doc fmt
+.PHONY: all install clean clean-cargo-lock clean-datasets datasets doc fmt
 
 WORKSPACE_DIR = $(abspath ..)
+BIN_DIR ?= /usr/local/cargo/bin
 include $(WORKSPACE_DIR)/common.mk
 
+
 all: crates/sdk/wasm-checker/bin/wasm-checker
 	cargo build $(PROFILE_FLAG)
 
+install: all
+	# create full names
+	cp -f target/$(PROFILE_PATH)/freestanding-execution-engine $(BIN_DIR)/freestanding-execution-engine
```

Comment:
Since you're not changing the name you could shorten these commands:
```
cp -f target/$(PROFILE_PATH)/freestanding-execution-engine $(BIN_DIR)/
```
etc